dispatcher/backend/src/tests/integration/routes/workers/test_worker.py

```diff
@@ -6,7 +6,7 @@
 class TestWorkersCommon:
     def test_build_workers_whitelist(self, workers):
         whitelist = build_workers_whitelist()
-        assert len(whitelist) == len(workers)
+        assert len(whitelist) == 36
```

Comment:
- ideally, we'd have a variable with the expected number
- static value in a test should be explained. How can next developer looking be sure about this or update it should fixture changed?
- don't change a test without mentioning is clearly; otherwise it looks like you broke a test and entered whatever value made the test pass witout fixing the issue 😅

Reply:
Agreed. Just to confirm my understanding, where did you expect me to mention it? In the PR, right?